### PR TITLE
mac80211: ath11k: restore 160MHz support

### PR DESCRIPTION
**Pull request by robimarko**
Recent ath11k sync introduced a regression causing 80+80 and 160MHz to stop being advertised and thus not selectable due to the respective feature flags being cleared.

So, until we get answers upstream to what was the reasoning behind this and it gets fixed, lets just remove the flag clearing to reanable 160MHz.

Fixes: 789a0bac3535 ("mac80211: ath11k: sync with ath-next")

### CODE REVIEW

**Comment by Soromeister** [comment]
If this gets approved for merging and is included in a new SNAPSHOT build, I can test on an AX9000 and report back.

**Review by Soromeister — APPROVED** [review]
Not sure why add a new `.patch` file instead of changing the existing `package/kernel/mac80211/patches/ath11k/0039-wifi-ath11k-push-MU-MIMO-params-from-hostapd-to-hard.patch`. 

@robimarko, are you sure this is the only change that needs to be done? Shouldn't the below also be adjusted?

- Lines 52-63 from `package/kernel/mac80211/patches/ath11k/0040-wifi-ath11k-move-HE-MCS-mapper-to-a-separate-functio.patch` seem to have been moved up to the **6**GHz declaration. It used to be under `ath11k_mac_copy_he_cap` and now it's under `ath11k_mac_set_hemcsmap`.
- The `package/kernel/mac80211/patches/ath11k/0041-wifi-ath11k-generate-rx-and-tx-mcs-maps-for-supporte.patch` has changed a bunch of declarations from `band_cap->he_mcs` to `rxmcs_map`. I suspect this has only to do with TX power adjustments but I thought to give it a shout-out here too.

**Comment by Soromeister** [comment]
All checks have passed. Not sure why it requires 6 reviews before merging but I hope this gets through. Thanks @robimarko !

**Comment by champtar** [comment]
@Soromeister for this repo maintainers don't merge via GitHub, and as it's not possible to disable merging, requiring 6 approve prevent accidental use of the merge button

**Comment by dick902** [comment]
Tested with Xiaomi AX9000 / Xiaomi AX3600 / Dynalink DL-WRX36.
All model's 5G 160MHz had been restore with this patch.

**Review by dick902 — APPROVED** [review]

**Comment by zxlhhyccc** [comment]
Devices such as AX6, AX3600 and AX9000 support 6G band. Why does the firmware not have this band?

**Comment by dick902** [comment]
> Devices such as AX6, AX3600 and AX9000 support 6G band. Why does the firmware not have this band?

The devices you mentioned were never supported 6G band as the required hardware did not existing inside those devices

**Comment by qrekkwijjtx54** [comment]
I ran into the same problem. I hope this request will be merged as soon as possible.

**Review by vitorcunha — APPROVED** [review]
Good workaround, revert when (*if*) upstream fixes the issue

**Pull request merged**
